detect_secrets/pre_commit_hook.py

```diff
@@ -113,7 +113,7 @@ def main(argv=None):
         pretty_print_diagnostics_for_verified_non_audited(verified_non_audited)
         return 2
 
-    # check if there are non-audited secrets
+    # check if there are unaudited secrets
     if args.fail_on_unaudited:
         non_audited = get_non_audited_secrets_from_baseline(
```

Comment:
How about this instance  of "non_auditied". and "get_non_auditied_secrets_from_baseline"

Reply:
I've refactored all remaining `non_audited` references to `unaudited`: https://github.com/IBM/detect-secrets/pull/91/commits/2430aebbf67ba36643e1bbe91a13791ddb07e0af